test/functional/unified-spec-runner/entities.ts

```diff
@@ -51,15 +53,8 @@ export type CmapEvent =
   | ConnectionCheckedInEvent
   | ConnectionPoolClearedEvent;
 
-function serverApiConfig() {
-  if (process.env.MONGODB_API_VERSION) {
-    return { version: process.env.MONGODB_API_VERSION };
-  }
-}
-
 function getClient(address) {
-  const serverApi = serverApiConfig();
-  return new MongoClient(`mongodb://${address}`, serverApi ? { serverApi } : {});
+  return new MongoClient(`mongodb://${address}`, getEnvironmentalOptions() as MongoClientOptions);
```

Comment:
The helper is defined in a js file so the return type can't go there unfortunately. I remember needing to put that type assertion because I was getting a ts error, but I just tried removing it and it's now compiling with no complaints. :shrug: